### ISSUE TITLE
Loss graph issue

### ISSUE THREAD

**Issue by giordyb** [mlflow/mlflow#221]
### System information
Centos 7.5 running mlflow in a container based on the python:3 image.
- **Have I written custom code (as opposed to using a stock example script provided in MLflow)**:
No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Centos 7
- **MLflow installed from (source or binary)**:  pip install mlflow
- **MLflow version (run ``mlflow --version``)**: 0.3.0
- **Python version**:  3.6.6
- **npm version (if running the dev UI):
- **Exact command to reproduce**:

### Describe the problem
I'm tracking the loss of a yolo3 training on keras with a mlflow server running on docker.  Even though the data looks right the plot is jumping all over the place. 

this is the plot that is showing in the gui:

![mlflow](https://user-images.githubusercontent.com/13320961/43482224-68b8836a-9508-11e8-9730-a4c0534fb256.jpg)

this is how the data is recorded in the uuid/metrics/loss file
```
0 29.311828909147355
0 10.147230804647718
0 8.57937268336614
0 5.942326284334773
0 4.77184491017035
0 4.335642215609551
0 4.012216826095467
0 3.9196282218041874
0 3.3551731505706197
0 3.4959531473829633
0 3.2127973465479553
0 3.2992866287203064
0 3.0219391981902577
0 2.924037829098247
0 2.7108263870009353
0 2.3252671778627803
```

### Source code / logs
this is how i'm using the mlflow.log_metric() in the training script:

```
class LossHistory(keras.callbacks.Callback):

        def on_epoch_end(self, batch, logs={}):
            mlflow.log_metric("loss", float(str(logs.get('loss'))))
            mlflow.log_metric("yolo_layer_1_loss", float(
                str(logs.get('yolo_layer_1_loss'))))
            mlflow.log_metric("yolo_layer_2_loss", float(
                str(logs.get('yolo_layer_2_loss'))))
            mlflow.log_metric("yolo_layer_3_loss", float(
                str(logs.get('yolo_layer_3_loss'))))
```
it seems to be happening only when there are more than 9 data points.

Nevermind just saw that it's a duplicate of #45

**Comment by mateiz** [comment]
Yup, this should be fixed by PR #204. Try it out and close the issue if it's fixed.

**Comment by giordyb** [comment]
I built from source (master branch) and I can confirm that the issue has been fixed.

**Comment by kepler** [comment]
I can confirm it's working in version `0.4.0`, already released in PyPI.

**Comment by aarondav** [comment]
Closing as fixed.
